src/react/utils/hooks.ts

```diff
@@ -199,3 +200,14 @@ export const useAccounts = () => {
     (account) => account.Name !== 'scality-internal-services',
   );
 };
+
+export const useInvalidAccountsQuery = () => {
+  const users = useSelector(
+    (state: AppState) => state.configuration.latest.users,
+  );
+  const queryClient = useQueryClient();
+  const token = useSelector((state: AppState) => state.oidc.user?.access_token);
+  useEffect(() => {
+    queryClient.invalidateQueries(['WebIdentityRoles', token]);
+  }, [users.length]);
```

Comment:
I'm not very sure to understand the intent of this code. We should invalidate WebIdentityRoles when we delete an account or create a new one, but for this we should probably not based the invalidation on the configuration overlay as we are studying it's removal

Reply:
That's right we should probably invalidate WebIdentityRoles cache based on the local storage of the accountID.
https://github.com/scality/zenko-ui/pull/334/commits/0129c206bfe2b2db6840033ad8030efeebf2fb5f

Reply:
Sorry I still feel it is a bit overcomplicated/not really optimised. To me having the cache invalidation next to the action (creation or deletion) would be preferable